_spinnaker/operator.md

```diff
@@ -56,7 +78,12 @@ To install Operator in basic mode, run:
 $ kubectl apply -n <op_namespace> -f deploy/operator/basic
 ```
 
-`op_namespace` is the namespace where you want the operator to live.
+`<op_namespace>` is the namespace where you want the operator to live. To monitor installation, run the following command:
```

Comment:
Wrong command. This would be to monitor a Spinnaker install. I'd delete line 81 ("To monitor..." up to the end of the bash command.